pkg/frontend/transport/handler.go

```diff
@@ -53,13 +53,15 @@ type HandlerConfig struct {
 	LogQueryRequestHeaders flagext.StringSliceCSV `yaml:"log_query_request_headers" category:"advanced"`
 	MaxBodySize            int64                  `yaml:"max_body_size" category:"advanced"`
 	QueryStatsEnabled      bool                   `yaml:"query_stats_enabled" category:"advanced"`
+	ResponseStreamTimeout  time.Duration          `yaml:"response_stream_timeout" category:"experimental"`
```

Comment:
Have you considered to name the config parameter to be specific for active series endpoints? Reason is that if in the future we add streaming response to other endpoints, I would argue a 10m default timeout for them may be very high and undesiderable.

Reply:
Given this looks applied only to write timeout, we could rename it to "active-series-write-timeout" to be more specific.